ci_framework/playbooks/05-build-operators.yml

```diff
@@ -23,6 +23,10 @@
           ansible.builtin.command: |
             oc registry login --insecure=true
 
+        - name: CRC project create
+          ansible.builtin.command: |
+            oc new-project openstack
```

Comment:
We can replace openstack with ``{{ cifmw_install_yamls_defaults['NAMESPACE'] }}`